games/tomsadventure/index.html

```diff
@@ -5,7 +5,7 @@
 		<meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi" />
 	</head>
 	<link rel="stylesheet" type="text/css" href="../../css/lcdstyle.css">
-	<script type="text/javascript" src="../../bin/platform.min.js"></script>
+	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/platform/1.3.6/platform.min.js"></script>
```

Comment:
I suppose you have platform.min.js on your server. Instead of tracking it in git as well, thought it would be easier to add a CDN hosted version here, so running it locally wouldn't break.